www/js/settings.json

```diff
@@ -74,16 +74,24 @@
             "step": 1
           }
         }
-      },
-      "sunder-gutka": {
-        "title": "Sundar Gutka",
-        "type": "range",
+      }
+    }
+  },
+  "toolbar": {
```

Comment:
would it be better if this was closer to fields to display/display options? might not matter.